src/components/Navbar/index.spec.tsx

```diff
@@ -264,10 +152,40 @@ describe(__filename, () => {
       expect(root.find(VersionChooser)).toHaveLength(0);
     });
 
-    it('does not render a link to the legacy file viewer', () => {
-      const root = render();
+    it('renders a link to the legacy file viewer in the browse page', () => {
+      const reviewersHost = 'https://example.com';
+      const path = '/some/file/path/file.js';
+      const store = configureStore();
+      const versionId = 12345;
+      store.dispatch(versionsActions.setCurrentVersionId({ versionId }));
+
+      const root = render({ reviewersHost, store, path });
+
+      expect(root.find(`.${styles.legacyLink}`)).toHaveProp(
+        'href',
+        `${reviewersHost}/${process.env.REACT_APP_DEFAULT_API_LANG}/firefox/files/browse-redirect/${versionId}/?file=${path}`,
+      );
+    });
+
+    it('renders a link to the legacy diff viewer in the compare page', () => {
```

Comment:
Nothing in this test case makes me think that we are in the compare page. Shouldn't it be something like: `...file viewer when both the currentVersionId and baseVersionId are set`?